### PR TITLE
Add Generate Offline Map (Basemap by Reference) sample

### PR DESCRIPTION
**Pull request by philium**

### CODE REVIEW

File: arcgis-ios-sdk-samples/Maps/Generate offline map (basemap by reference)/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Generate offline map with local basemap
```
Comment (saratchandrakarumuri) [inline arcgis-ios-sdk-samples/Maps/Generate offline map (basemap by reference)/README.md]: Sample name and title in readme doesn't match

Comment (philium) [reply arcgis-ios-sdk-samples/Maps/Generate offline map (basemap by reference)/README.md]: Thanks. I'll update the readme to match what is used by the app.

**Comment by philium** [comment]
@njarecha Could you please re-approve this PR at your earliest convenience?

**Comment by philium** [comment]
@njarecha @saratchandrakarumuri Thank you!

**Pull request merged**